src/script/backport-create-issue

```diff
@@ -53,13 +53,17 @@ tracker2tracker_id = {}
 version2version_id = {}
 
 def usage():
-    logging.error("Command-line arguments must include either a Redmine key (positional "
-                  "argument) or a Redmine username and password (via --user and --password)")
+    logging.error("Command-line arguments must include either a Redmine key (--key) "
+                  "or a Redmine username and password (via --user and --password). "
+                  "Optionally, a comma-separated list of issue numbers can be given "
+                  "via a positional argument. In the absence of a positional argument, "
+                  "the script will loop through all issues in Pending Backport status.")
     exit(-1)
 
 def parse_arguments():
     parser = argparse.ArgumentParser()
-    parser.add_argument("key", nargs='?', help="Redmine user key")
+    parser.add_argument("issuespec", nargs='?', help="Issue number")
```

Comment:
What about more than one issue? i.e. `nargs='+'`?

Reply:
comma-separated string (convenient, because it can be fed directly to `redmine.issue.filter`)

Reply:
I can add heuristics. It'll be nicer if we can do stuff like `20301 20473 25342` and easy to convert that into a comma-separated string.